syncd/syncd_notifications.cpp

```diff
@@ -92,19 +92,122 @@ void redisPutFdbEntryToAsicView(
+                    {"id":"SAI_FDB_ENTRY_ATTR_PACKET_ACTION","value":"SAI_PACKET_ACTION_FORWARD"}
+                ]
+            }]
+            */
+            SWSS_LOG_NOTICE("received a flush all fdb event");
+            std::string pattern = ASIC_STATE_TABLE + std::string(":SAI_OBJECT_TYPE_FDB_ENTRY:*");
+            for (const auto &fdbkey: g_redisClient->keys(pattern))
+            {
+                SWSS_LOG_DEBUG("remove fdb entry %s for SAI_FDB_EVENT_FLUSHED",fdbkey.c_str());
+                g_redisClient->del(fdbkey);
+            }
+        }
+        else if( port_oid && !vlan_id )
+        {
+            /* we got a flush port fdb event here        */
+            /* not supported yet, this is a place holder */
+            /* example of a flush port fdb event         */
+            /*
+            [{
+            "fdb_entry":"{
+                \"bridge_id\":\"oid:0x0\",
+                \"bridge_type\":\"SAI_FDB_ENTRY_BRIDGE_TYPE_1Q\",
+                \"mac\":\"00:00:00:00:00:00\",
+                \"switch_id\":\"oid:0x21000000000000\",
+                \"vlan\":\"0\"}",
+            "fdb_event":"SAI_FDB_EVENT_FLUSHED",
+                "list":[
+                    {"id":"SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID","value":"oid:0x3a0000000009cf"},
+                    {"id":"SAI_FDB_ENTRY_ATTR_TYPE","value":"SAI_FDB_ENTRY_TYPE_DYNAMIC"},
+                    {"id":"SAI_FDB_ENTRY_ATTR_PACKET_ACTION","value":"SAI_PACKET_ACTION_FORWARD"}
+                ]
+            }]
+            */
+            SWSS_LOG_ERROR("received a flush port fdb event, unsupported");
+        }
+        else if(!port_oid && vlan_id)
+        {
+            /* we got a flush vlan fdb event here        */
+            /* not supported yet, this is a place holder */
+            /* example of a flush vlan event             */
+            /*
+            [{
+            "fdb_entry":"{
+                \"bridge_id\":\"oid:0x0\",
+                \"bridge_type\":\"SAI_FDB_ENTRY_BRIDGE_TYPE_1Q\",
+                \"mac\":\"00:00:00:00:00:00\",
+                \"switch_id\":\"oid:0x21000000000000\",
+                \"vlan\":\"1000\"}",
+            "fdb_event":"SAI_FDB_EVENT_FLUSHED",
+                "list":[
+                    {"id":"SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID","value":"oid:0x0"},
+                    {"id":"SAI_FDB_ENTRY_ATTR_TYPE","value":"SAI_FDB_ENTRY_TYPE_DYNAMIC"},
+                    {"id":"SAI_FDB_ENTRY_ATTR_PACKET_ACTION","value":"SAI_PACKET_ACTION_FORWARD"}
+                ]
+            }]
+            */
+            SWSS_LOG_ERROR("received a flush vlan fdb event, unsupported");
+            
+        }
+        else if(port_oid && vlan_id)
+        {
+            SWSS_LOG_ERROR("received a flush port in a vlan fdb event, unsupported");
+        }
+        else
+        {
+            SWSS_LOG_ERROR("received a unknown flush fdb event, unsupported");
+        }
```

Comment:
those 2 erros can be combined in 1 error message and print port_oid and vlan_id

Reply:
revised.